pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -92,6 +92,7 @@ style_min: false
 - js_locale = request.locale.to_s.downcase.tr('-', '_')
 %script{src: asset_path("js/#{js_locale}/tutorialExplorer_locale.js")}
 %script{src: minifiable_asset_path("js/tutorialExplorer.js")}
+%script{src: "https://cdn.optimizely.com/js/400912536.js"}
```

Comment:
Might be worth moving to the top of the page so that it can kick in faster, per their recommendations?